sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuTransitionOverrides.scala

```diff
@@ -407,7 +406,12 @@ class GpuTransitionOverrides extends Rule[SparkPlan] {
     }
   }
 
-  private def insertHashOptimizeSorts(plan: SparkPlan): SparkPlan = {
+  // Find  GpuDataWritingCommandExec(......(GpuHashJoin)),
+  //       GpuDataWritingCommandExec(......(GpuHashAggregateExec))
+  // from top to bottom and insert sort to optimize the file size.
+  // such as: GpuDataWritingCommandExec(GpuProject(GpuHashAggregateExec))
```

Comment:
This comment is no longer accurate, suggest something like:
```suggestion
  // If a GPU hash-based operation, such as GpuHashJoin or GpuHashAggregateExec,
  // is followed eventually by a data writing command without an intermediate node
  // changing the sort order, insert a sort to optimize the output file size.
```

Reply:
done